Jenkinsfile.release

```diff
@@ -7,9 +7,9 @@ ansiColor('xterm') {
       parameters([
         string(name: 'version',
           defaultValue: '',
-          description: """Community Release Version (ex. 1.5.2).
+          description: """Community Release Version (ex. 1.6.322-2bf46b341).
```

Comment:
I think that community version is without git hash. As you noted, maybe just the property/description is not correct

Reply:
Done. Thanks for the feedback.